2018 Robot/src/org/usfirst/frc/team4999/utils/MoPrefs.java

```diff
@@ -0,0 +1,64 @@
+package org.usfirst.frc.team4999.utils;
+
+import edu.wpi.first.wpilibj.Preferences;
+
+public class MoPrefs {
+	
+	private static Preferences prefs = Preferences.getInstance();
+	
+	private static final double AUTO_SPEED = 0.25; 				// Maximum speed the robot will travel at in autonomous
+	private static final double DRIVE_ENC_TICKS = 562.666378; 	// Number of encoder ticks per meter of travel
+	private static final double LIFT_ENC_TICKS = 500; 			// Number of lift encoder ticks per meter of travel
+	private static final double FALLBACK_AUTO_TIME = 3; 		// Time, in seconds, the robot will drive for when in fallback auto mode
+	private static final double FALLBACK_AUTO_DISTANCE = 3.5; 	// Distance the robot will drive for, in meters, in fallback auto mode
+	private static final double TILT_RANGE = 2; 				// Degrees robot will tilt off-axis before attempting to stabilize
```

Comment:
How many degrees of tilt do we get due to the west coast drive?
How many degrees of tilt do we get driving onto the platform?